MovieApp/Views/HomeScreen/MovieCategoryView.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  MovieCategoryView.swift
+//  MovieApp
+//
+//  Created by Nikola Đokić on 13.04.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class MovieCategoryView: UIView{
+    
+    private var categoryTitle: UILabel!
+    private var categorySelectButton1: UIButton!
+    private var categorySelectButton2: UIButton!
+    private var categorySelectButton3: UIButton!
+    private var categorySelectButton4: UIButton!
+    private var underlineImage: UIImageView!
+    private var buttonPressed = 1
+    
+    init(categoryTitleText: String, buttonText1: String, buttonText2: String, buttonText3: String, buttonText4: String){
```

Comment:
Bad solution because it limits the functionality to 4 buttons. It would be a lot better if you had a horizontal stack view that you can fill with as many buttons as you need. Your init can receive a String array of button titles and you could then simply create the buttons with those titles. The tag could be the index in the received array